main.go

```diff
@@ -135,6 +142,7 @@ type parseCmd struct {
 	Source string `arg:"" help:"Source file. Default stdin" default:"-"`
 }
 
+// Run executes the `evy run` command according to Kong API.
```

Comment:
no I can follow along i think. Happy to change to your wording, I think I agree, it's better.

Reply:
Though similar to the comment on slack, imo `evy run` is the bit the reader would care most about and should be mentioned first.

Reply:
ok. well I will change this now because i think our wording is better but I would have still preferred for `evy run` to be mentioned before Kong.
I understand your ramble, but in this case this is what it is for I (executing `evy run` via kong).

Reply:
ok, how about "Run implements the `evy run` CLI command, called by the Kong API"?